File: py3/removeElement.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:

        i = 0;
        while(i < len(nums)):
            if(nums[i] == val):
                print("i:",i, nums)
                nums.remove(val);
                print("new arr = ", nums)
                i = -1;
            i+=1

        return len(nums)


S = Solution();
a = [1,2,2]

print(S.removeElement(a, 1))

File: removeElement.js
var removeElement = function(nums, val) {

    for(let i =0; i<nums.length; i++){
        if(nums[i] === val){
            nums.splice(i,1);
            i = 0;
        }
    }
    if(nums[0] === val){
        nums.splice(0,1);
    }
    return nums.length;


};
